Makefile

```diff
@@ -23,4 +27,16 @@ vendor:
 test: vendor
 	go test ./pkg/...
 
+images: asb-image
+
+asb-image:
```

Comment:
Since you already have a branch for a more long term build process, I decided not to make this perfect. It builds what we have today, and I'm all for moving this to a better process.